src/test/java/com/futurecollars/OOPTask4/ConvertTest.java

```diff
@@ -0,0 +1,23 @@
+package com.futurecollars.OOPTask4;
+
+import org.junit.jupiter.api.Test;
+
+import org.junit.jupiter.api.Assertions;
+
+class ConvertTest {
+
+    @Test
+    void shouldCheckIfConvertsDouble() {
+        double minutes = 5.3;
+        double seconds = minutes * 60;
+        Assertions.assertEquals(318, seconds);
```

Comment:
ale co ty tutaj testujesz?:D czemu test testuje... test?

Reply:
przecież jak ja teraz zmienię, przez pomyłkę/celowo twój kod w convertion to twój test tego kompletnie nie wykryje ;)